### ISSUE TITLE
Ignore test_tvu_exit

### ISSUE THREAD

**Issue by mvilnes** [solana-labs-community/solana#110]
`test_tvu_exit continues` to be flaky and doesn't really seem to be even testing anything interesting. Ignore it

**Comment by mvilnes** [comment]
The final straw: https://buildkite.com/solana-labs/solana/builds/24472#3c6bc3df-ec18-4165-8d33-c7b9f0a63b52

**Comment by CrieasofCarrots** [comment]
Unable to reproduce so I'd say my hypothesis was definitely wrong. Maybe we can chalk this up to old code / old bug

**Comment by CrieasofCarrots** [comment]
Looks like that failure should be a lot easier to debug than the more complex test that fails next.

**Comment by mvilnes** [comment]
Yeah and yet it's been failing for months with nobody stepping up to fix
